### DISCUSSION TITLE
Mantine Charts valueFormatter RightYAxis

### DISCUSSION THREAD

**Discussion by kerkhoff-adobe** [mantinedev/mantine#6893] [Feature requests]
Can we use a different formatter for the RightYAxis? For example when displaying data traffic and number of requests. With the current formatter, both are transformed to TB. Would be great if the valueFormatter function would also provide the datafield name or axis so data can returned based on that.

Thanks!

**Comment by rtivital** [comment]
You can pass props down to recharts YAxis component with `rightYAxisProps` prop
